lib/school.rb

```diff
@@ -0,0 +1,35 @@
+class School
+  attr_reader:start_time,
+             :hours_in_school_day,
+             :student_names
+  def initialize(start_time, hours_in_school_day)
+    @start_time = start_time
+    @hours_in_school_day = hours_in_school_day
+    @student_names = []
+  end
+
+  def add_student_name(name)
+    @student_names << name
+  end
+
+  def end_time
+    (@start_time.to_i + @hours_in_school_day).to_s + ":00"
+  end
+
+  def is_full_time?
+    @hours_in_school_day > 4
```

Comment:
More of an advanced concept but you can probably assign the value `4` here to a constant.

So something like `FULL_TIME_HOURS = 4` somewhere in your class scope